serverless.js

```diff
@@ -90,7 +90,7 @@ class Backend extends Component {
       description: inputs.description || 'A function for the Backend Component',
       memory: inputs.memory || 896,
       timeout: inputs.timeout || 10,
-      runtime: 'nodejs8.10',
+      runtime: inputs.runtime || 'nodejs10.x',
```

Comment:
@RodrigoEspinosa we should really throw an error if it's not a node runtime, cause only node is supported due to the shim

Reply:
Good idea, I'll make sure to include it today, thanks!

Reply:
Would something like this make sense, or making the supported runtimes dynamic isn't necessary at this point?

```js
// Get a list of accepted runtimes
const acceptedRuntimes =
  this.types().functions.default.inputs.find((input) => input.name === 'code').runtimes || []

// Check if the runtime from the input is not accepted
if (inputs.runtime && !acceptedRuntimes.findIndex(inputs.runtime)) {
  throw new Error(
    `The runtime ${inputs.runtime} is not accepted. Currently only ${acceptedRuntimes.join(
      ', '
    )} are supported.`
  )
}
```

Reply:
Imo it's not necessary at this point. The types that you're referencing to is still experimental and we don't currently rely on it.

Reply:
Sorry for the delay, it's there now. Let me know if you want more changes there.

Reply:
Thanks @RodrigoEspinosa ... but I don't see the error handling logic! 🤔 ... what would happen if I specify a runtime other than node?

Reply:
Whoops! My apologies. Should be ok now, let me know if you the message to say something else.